### PR TITLE
Fixes #14759: debian server 5.1 packages won't build  

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/14759

### CODE REVIEW

File: rudder-server-relay/debian/rules [hunk 1]
```diff
@@ -8,14 +8,7 @@ override_dh_auto_build:
...
 	mv rudder-sources-*/* BUILD/
-	dh_auto_build -- -C BUILD/rudder/relay/sources/
-
-override_dh_auto_install:
-	dh_auto_install -- -C BUILD/rudder/relay/sources/
-
-override_dh_auto_clean:
-# clean/distclean would destroy the make localdepends of package builder, cleanup this code before enabling
-#	dh_auto_clean -- -C BUILD/rudder/relay/sources/
+	dh_auto_build
```
Comment (amousset) [inline rudder-server-relay/debian/rules]: so much better!

File: rudder-server-root/Makefile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Used on debian to redirect calls to final Makefile so dh auto commands work
+%:
+	make -C BUILD/rudder/relay/sources/ $@
```
Comment (amousset) [inline rudder-server-root/Makefile]: wrong path

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Pull request merged**